firstvoices/backend/views/search/base_search_views.py

```diff
@@ -263,8 +263,16 @@ def get_pagination_params(self):
         """
         default_page_size = self.paginator.get_page_size(self.request)
 
-        page = int(self.request.GET.get("page", 1))
-        page_size = int(self.request.GET.get("pageSize", default_page_size))
+        valid_page_number = self.paginator.override_invalid_number(
+            self.request.GET.get("page", 1)
+        )
+        page = valid_page_number
```

Comment:
maybe we can directly equate these 2 variables to the methods ?
like 
`page = self.paginator.override_invalid_number(self.request.GET.get("page", 1))`
and similarly for `page_size`